Squirrel/Controllers/LanguageController.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.AspNetCore.Localization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Squirrel.Controllers
+{
+    [ApiController]
+    [Route("api/[controller]/[action]")]
+    public class LanguageController : Controller
+    {
+        [HttpGet]
+        public IActionResult SetLanguage(string culture)
+        {
+            Response.Cookies.Append(key: CookieRequestCultureProvider.DefaultCookieName,
```

Comment:
What is the reason for the business layer to know anything about the presentation layer? I'd like to discuss this architectural approach